MicroProfile-Fault-Tolerance/tck-runner/src/test/resources/tck-suite2.1-debug.xml

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
```

Comment:
Was ist das?

A debug file to test only BulkheadAsynchTest?

Reply:
Yes and no. When working on the TCK you'll always need one config to debug particular issues. I thought why create a fresh file every time. The context isn't important here. Just that there is yet another config one can mess with. But I could also delete it. Not really important.